docs/go-storage/internal/object-lazy-stat.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Object Lazy Stat
+---
+
+## Introduction
+
+We use `types.NewObject(client, done)` to create a new object with client in services. 
+
+```go
+func NewObject(client Storager, done bool) *Object
+```
+
+**Parameters**
+
+client - the client in which `Object` is alive.
+
+done - marks whether `stat` for the object has been performed. `true` indicates `stat` has been performed, we don't need to `stat` for this object anymore.

```

Comment:
When will we create an object on which `stat` has been performed? I'm a little bit confused